robotidy/transformers/SplitTooLongLine.py

```diff
@@ -0,0 +1,92 @@
+from robot.api.parsing import (
+    ModelTransformer,
+    Token
+)
+
+
+EOL = Token(Token.EOL)
+CONTINUATION = Token(Token.CONTINUATION)
+
+
+class SplitTooLongLine(ModelTransformer):
+    def __init__(self, line_length: int = 120, split_on_every_arg: bool = False):
+        super().__init__()
+        self.line_length = line_length
+        self.split_on_every_arg = split_on_every_arg
+
+    def visit_KeywordCall(self, node):  # noqa
+        if all(line[-1].end_col_offset < self.line_length for line in node.lines):
+            return node
+        return self.split_keyword_call(node)
+
+    def split_keyword_call(self, node):
+        separator = Token(Token.SEPARATOR, self.formatting_config.space_count * ' ')
+        indent = node.tokens[0]
+
+        assignment = node.get_token(Token.ASSIGN)
+        keyword = node.get_token(Token.KEYWORD)
+
+        if assignment:
+            head = [indent, assignment, separator, keyword]
+        else:
+            head = [indent, keyword]
+
+        comments, tail, line = [], [], head
+
+        # Comments with separators inside them are split into
+        # [COMMENT, SEPARATOR, COMMENT] tokens in the AST, so in order to preserve the
+        # original comment, we need a lookback on the separator tokens.
+        last_separator = None
+
+        rest = node.tokens[node.tokens.index(keyword) + 1:]
+        for token in rest:
+            if token.type == Token.SEPARATOR:
+                last_separator = token
+            elif token.type in {Token.EOL, Token.CONTINUATION}:
+                continue
+            elif token.type == Token.COMMENT:
+                # AST splits comments with separators, e.g.
+                #
+                # "# Comment     rest" -> ["# Comment", "     ", "rest"].
+                #
+                # Notice the third value not starting with a hash - that's what this
+                # condition is about:
+                if not str(token).startswith('#'):
+                    # -2 because -1 is the EOL
+                    comments[-2].value += last_separator.value + token.value
+                else:
+                    comments += [indent, token, EOL]
+            elif token.type == Token.ARGUMENT:
+                if token.value == '':
+                    token.value = '${EMPTY}'
```

Comment:
why it's needed? Is this added so it's easier to see arguments with empty value?